<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1407</width>
    <height>1097</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="word_list">
    <property name="geometry">
     <rect>
      <x>113</x>
      <y>123</y>
      <width>1111</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_create_csv">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>660</y>
      <width>601</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>Seçili Veritabanından CSV Dosyası Oluştur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_delete_word">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>660</y>
      <width>601</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Selected Word</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>431</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Seçili Kelime Listesi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_database_list">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1407</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
